SingularityUI/app/components/taskDetail/TaskDetail.jsx

```diff
@@ -477,7 +478,12 @@ class TaskDetail extends Component {
     return (
       <div className="task-detail detail-view">
         {this.renderHeader(cleanup)}
-        <TaskAlerts task={this.props.task} deploy={this.props.deploy} pendingDeploys={this.props.pendingDeploys} />
+        <TaskAlerts
+          task={this.props.task}
+          deploy={this.props.deploy}
+          pendingDeploys={this.props.pendingDeploys}
+          onViewHealthchecks={() => this.setState({ expandedHealthchecks: true })}
```

Comment:
It feels a little weird to have a `onViewHealthchecks` prop. That prop is really specific. This implementation also doesn't seem to auto-expand the section when using the url directly or refreshing the page.